README.md

```diff
@@ -21,261 +20,296 @@ limitations under the License.
 [![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/projectmonai/monai-label/10)](https://dev.azure.com/projectmonai/monai-label/_build?definitionId=10)
 [![codecov](https://codecov.io/gh/Project-MONAI/MONAILabel/branch/main/graph/badge.svg)](https://codecov.io/gh/Project-MONAI/MONAILabel)
 
+MONAI Label is an intelligent open source image labeling and learning tool that enables users to create annotated datasets and build AI annotation models for clinical evaluation. MONAI Label enables application developers to build labeling apps in a serverless way, where custom labeling apps are exposed as a service through the MONAI Label Server.
+
 MONAI Label is a server-client system that facilitates interactive medical image annotation by using AI. It is an
 open-source and easy-to-install ecosystem that can run locally on a machine with single or multiple GPUs. Both server
 and client work on the same/different machine. It shares the same principles
-with [MONAI](https://github.com/Project-MONAI). Refer to full [MONAI Label documentations](https://docs.monai.io/projects/label/en/latest/index.html) for more details.
+with [MONAI](https://github.com/Project-MONAI).
 
+Refer to full [MONAI Label documentations](https://docs.monai.io/projects/label/en/latest/index.html) for more details or check out our [MONAI Label Deep Dive videos series](https://www.youtube.com/playlist?list=PLtoSVSQ2XzyD4lc-lAacFBzOdv5Ou-9IA).
 
-- [MONAI Label Demo: Sample Apps](#sample-apps-in-monailabel)
-- [Highlights and Features](#hightlights-and-features)
-- [Installation and Usage](#installation)
-  - [MONAI Label Server](#current-stable-version)
-  - [Visualization Tools Guide](#visualization-tools)
-  - [Plugin Guide](#plugins)
-- [MONAI Label Integration Tutorials](https://github.com/Project-MONAI/tutorials/tree/main/monailabel)
-- [Contributing Guide and Communities](#contributing)
 
-## Sample Apps in MONAILabel
 
-![image](https://raw.githubusercontent.com/Project-MONAI/MONAILabel/main/docs/images/sampleApps_index.jpeg)
+### Table of Contents
+- [Overview](#Overview)
+  - [Highlights and Features](#Highlights-and-Features)
+  - [Supported Matrix](#Support-Matrix)
```

Comment:
Hi @zephyrie , could you double confirm if the links/references are working, I see some of the reference links can not redirect to sections. Thank you!